pr_4/javascript/random_samples/cross_site_request_forgery.js

```
@@ -0,0 +1,32 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+// {ex-fact rule=cross-site-request-forgery@v1.0 defects=1}
+var express = require('express')
+var app = express()
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)